lib/rory/application.rb

```diff
@@ -1,6 +1,9 @@
 require 'pathname'
 require 'logger'
 require 'rory/route_mapper'
+require 'rack/commonlogger'
+require_relative '../rory/request_parameter_logger'
+
```

Comment:
It's fine to use require_relative inconsistently for now - we can refactor this later - but at least don't have unnecessary path traversal.  Just `require_relative 'request_parameter_logger'` should do the trick.